wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -547,9 +546,13 @@ function show_dictionary_search() {
  * Function executed by AJAX to remotely get data from Oxford Dictionary
  * */
 function search_oxford_dictionary(){
+    $result = wp_remote_get( esc_url('https://www.oxfordlearnersdictionaries.com/definition/english/' . sanitize_title_with_dashes( $_POST['word']) ) );
 
-    $result = wp_remote_get( 'https://www.oxfordlearnersdictionaries.com/definition/english/' . sanitize_title_with_dashes( $_POST['word'] ) );
-    echo $result['body'];
+    if( is_wp_error( $result ) ) 
+        return;
+
+    $body = wp_remote_retrieve_body( $result );
+    echo $body;
```

Comment:
Could you try running `$body` through `wp_kses_post()` first. It might completely break it but let's try. Let me know the result.
https://developer.wordpress.org/reference/functions/wp_kses_post/

Reply:
I'll paste here what i get after running it: 

window.geofeed = function(geoFeedData) { window.geofeedData = geoFeedData; window.isGDPRMode = function() { return "true" === "true"; }; window.isCCPAMode = function() { return (geoFeedData.country == "US" && geoFeedData.state == "CA"); }; if (window.isCCPAMode()) { var otCCPAiab = document.createElement('script'); otCCPAiab.setAttribute('src', 'https://cdn.cookielaw.org/opt-out/otCCPAiab.js'); otCCPAiab.setAttribute('charset', 'UTF-8'); otCCPAiab.setAttribute('ccpa-opt-out-ids', 'C0004'); otCCPAiab.setAttribute('ccpa-opt-out-geo', 'ca'); otCCPAiab.setAttribute('ccpa-opt-out-lspa', 'false'); document.head.appendChild(otCCPAiab); } }; function OptanonWrapper() { } // var arrayFn = []; var dictCodesArr = new Array(); /*english * OALD9,OALD10 * free * false || true*/ dictCodesArr["english"] = { url : 'english', name : 'English', description : 'Search Oxford Advanced Learner\'s Dictionary', free: true }; /*american_english * OAAD,OALD9,OALD10 * free * false || true*/ dictCodesArr["american_english"] = { url : 'american_english', name : 'American English', description : 'Search Oxford Advanced American Dictionary', free: true }; /*academic * OLDAE * free * false || false*/ dictCodesArr["academic"] = { url : 'academic', name : 'Academic English', description : 'Search Oxford Learner\'s Dictionary of Academic English', free: false }; /*collocations * OCOLL * free * false || false*/ dictCodesArr["collocations"] = { url : 'collocations', name : 'Collocations', description : 'Search Oxford Collocations Dictionary', free: false }; /*practical-english-usage * PEU * free * false || false*/ dictCodesArr["practical-english-usage"] = { url : 'practical-english-usage', name : 'Practical English Usage', description : 'Search PEU index', free: false }; /*schulwoerterbuch_German-English * SWB * free * false || false*/ dictCodesArr["schulwoerterbuch_German-English"] = { url : 'schulwoerterbuch_German-English', name : 'German-English', description : 'Search Das Oxford Schulw\u00F6rterbuch', free: false }; /*schulwoerterbuch_English-German * SWB * free * false || false*/ dictCodesArr["schulwoerterbuch_English-German"] = { url : 'schulwoerterbuch_English-German', name : 'English-German', description : 'Search Das Oxford Schulw\u00F6rterbuch', free: false }; var sendGaData = function(data) { if(window.ga != null && OptanonActiveGroups.indexOf("C0002") != -1) { ga('send', data); } }; (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga'); ga('create', 'UA-13175707-1', { 'siteSpeedSampleRate': 10 }); ga('set', 'anonymizeIp', true); ga('send', 'pageview'); if(typeof iaw !== 'undefined') { iaw.que.push(function() { iaw.setGA(ga); }); }